pg_search/tests/pg_regress/sql/score_join_predicates.sql

```diff
@@ -148,7 +161,8 @@ SELECT
     a.name as author_name,
     paradedb.score(a.id) as author_score
 FROM authors a 
-WHERE a.name @@@ 'Rowling';
+WHERE a.name @@@ 'Rowling'
+ORDER BY a.id;
 
```

Comment:
it would be quite a lot easier to read these diffs if the `;` was on a line on its own, so adding new segments to a query changes 2 lines in the diff (removing the `;` in 151 and adding the _actual_ new thing in 164)